examples/scripts/utils.py

```diff
@@ -26,7 +27,8 @@ def make_tar(
     tarball_write_mode: str = "w:gz",
     working_dir: str | Path | None = None,
 ) -> Path:
-    """Archives a list of directories and files and compresses the archive.
+    """Archives a list of directories and files, flattening the directory struture
+    and then compresses the archive.
```

Comment:
The [Google-style of docstrings](https://google.github.io/styleguide/pyguide.html#38-comments-and-docstrings) suggests that the first line of a docstring should be a one-line summary with no wraparound. Additional information can be included in a paragraph that starts below the one-line summary. So I would rewrite the docstring like this:

```suggestion
    """Archive a list of directories and files.
    
    This implementation flattens the source directory structure so that all the files
    are placed in the root of the archive, and then the archive is compressed.
```